cadence/tests/scheduled_supervisor_test.cdc

```diff
@@ -912,11 +912,11 @@ fun testInsufficientFundsAndRecovery() {
     log("========================================")
 }
 
-access(all)
-fun main() {
-    setup()
-    testAutoRegisterAndSupervisor()
-    testMultiYieldVaultNativeScheduling()
-    testStuckYieldVaultDetectionLogic()
-    testInsufficientFundsAndRecovery()
-}
+// access(all)
```

Comment:
What is this? should we just remove?

Reply:
Addressed in [ca02657](https://github.com/onflow/FlowYieldVaults/pull/109/commits/ca02657f27c779160a6695ac21101ba05129abc9)